lab-cop/config/definitions/lab-cop.yml

```diff
@@ -0,0 +1,41 @@
+- scm:
+    name: lab-cop
+    scm:
+      - git:
+          url: https://github.com/ceph/teuthology.git
+          branches:
+            - ${sha1}
```

Comment:
@alfredodeza thank you 
yeah I already nuked the trigger section -- I plan to kick it manually for now. 
I'll get rid of this too